documentation/bolt.ditamap

```diff
@@ -75,5 +75,7 @@
         <topicref href="plan_functions.md" format="markdown"/>
         <topicref href="bolt_types_reference.md" format="markdown" />
         <topicref href="bolt_configuration_reference.md" format="markdown"/>
+        <topicref href="bolt_defaults_reference.md" format="markdown"/>
```

Comment:
I wonder if it makes sense to rename `Bolt configuration options` reference page to `bolt.yaml configuration options`. That'd be very slightly less confusing

Reply:
That page contains options for other files and will be part of the work for #1888